cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java

```diff
@@ -162,6 +162,7 @@ protected Map<String, CertAndKey> generateCcCerts(String namespace, String kafka
             subject.addDnsName(ccDnsGenerator.serviceDnsNameWithoutClusterDomain());
             subject.addDnsName(ccDnsGenerator.serviceDnsName());
             subject.addDnsName(CruiseControlResources.serviceName(kafkaName));
+            subject.addDnsName(node.podName());
```

Comment:
Wait ... are you adding it just randomly to all certificates? Why does CC need it?

Reply:
See below answer.